Migrate existing inputs to vuetify

rossicler opened the pull request:
## Proposed Changes

It was migrated the text inputs of the project to vuetify. #378 

## Type of Changes

What type of change this Pull Request brings to Falko?

- [ ] Bugfix
- [x] New Feature

## Checklist

- [x] This Pull Request has a significant name.
- [ ] The build is okay (tests, code climate).
- [x] This Pull Request mentions a related issue.

rossicler committed:
Changed login inputs to vuetify text-fields components

guilhermesiqueira committed:
Update vuetify version from 1.5.14 to 2.0.18 and migrating some buttons.

Co-authored-by: gabrielademoraes <gabrielademoraes10@gmail.com>

rossicler committed:
Fixed inputs at login form and changed inputs of register to Vuetify

rossicler committed:
Changed inputs to Vuetify of AddIssue, AddProject, EditProject, AddRelease and EditUserProfile

gabrielademoraes committed:
Migrating close button to vuetify.

Co-authored-by: guilhermesiqueira <guilhersiqueira@gmail.com>

guilhermesiqueira committed:
Merge branch 'devel' of https://github.com/falko-org/Falko into issue376-migrate-existing-buttons-to-vuetify

rossicler committed:
Changed inputs to vuetify of EditIssue, EditRelease, AddRetrospective and EditRetrospective

rossicler committed:
Updated version of vuetify

rossicler committed:
Removed outlined of inputs and fixed label position

lukassxp committed:
Changed inputs to vuetify of addSprint and editSprint

rossicler committed:
Resolved conflicts

rossicler committed:
Revert "Resolved conflicts"

This reverts commit 44e7e4f2c73e74c3087170bd4173ec5734e2f7e8.

rossicler committed:
Merge branch 'devel' into issue-378-vuetify-inputs

MatheusRich commented:
@rossicler It seems that you're updating all the buttons too, but it was also addressed on #379.

rossicler commented:
> @rossicler It seems that you're updating all the buttons too, but it was also addressed on #379.

Thanks for the reply, I reverted the merge commit of the buttons migration.

MatheusRich commented on src/components/Authentication/Register.vue:
```diff
@@ -129,12 +128,12 @@ export default {
 }
 
-#registerForm input {
+/* #registerForm input {
```
Is this still used?

MatheusRich commented on src/components/Retrospective/AddRetrospective.vue:
```diff
@@ -155,14 +161,14 @@ export default {
 <style scoped>
 
-textarea {
+/* textarea {
```
is this still being used?

MatheusRich commented on src/components/Retrospective/List.vue:
```diff
@@ -72,13 +72,13 @@ export default {
 
 <style scoped>
-input {
+/* input {
```
Same here

MatheusRich commented on src/components/Revision/ListRevision.vue:
```diff
@@ -66,13 +66,13 @@ export default {
 
 <style scoped>
-input {
+/* input {
```
Same here

MatheusRich reviewed (commented):
this seems to be good. Please fix the conflicts in `package.json`

rossicler committed:
Merge branch 'devel' into issue-378-vuetify-inputs

rossicler committed:
Deleted unused code

rossicler committed:
Merge branch 'issue-378-vuetify-inputs' of https://github.com/GCES-Falko-2019-2/Falko into issue-378-vuetify-inputs

rossicler commented:
> this seems to be good. Please fix the conflicts in `package.json`

Conflict resolved, also deleted the commented unused css.

MatheusRich commented:
@alaxalves @MartinsLucas can you guys take a look at this?
